src/mapproxyUtils/mapproxySeed.ts

```diff
@@ -57,7 +57,11 @@ export class MapproxySeed {
         throw new Error(`Date string must be 'ISO_8601' format: yyyy-MM-dd'T'HH:mm:ss, for example: 2023-11-07T12:35:00`);
       }
 
-      await this.writeMapproxyYaml(jobId, taskId);
+      const mapproxyConfig = await this.writeMapproxyYaml(jobId, taskId);
```

Comment:
should'nt it check "isRedisCache" before writing the file?

Reply:
it handles the overall mapproxyConfig existence, maybe I can't move the validation function into this writing mapproxy config...